examples/fabric-bridge-app/fabric-bridge-common/src/ZCLCallbacks.cpp

```diff
@@ -90,3 +91,13 @@ Protocols::InteractionModel::Status emberAfExternalAttributeWriteCallback(Endpoi
 
     return ret;
 }
+
+void MatterCommissionerControlPluginServerInitCallback()
```

Comment:
This callback is meant to be implemented by the SDK itself, to do whatever it needs to do for non-endpoint-specific cluster backend init, not applications.